docs/openstack/stop_openstack_services.md

```diff
@@ -64,6 +68,7 @@ there are no ongoing  operations.
 1- Connect to all the controller nodes.
 2- Stop the services.
 3- Make sure all the services are stopped.
+4- Repeat steps 1-3 for compute hosts (workloads running on dataplane will not be affected)
```

Comment:
If we want to stop those services everywhere, we should probably expand the snippet below to iterate over compute nodes too and not just `CONTROLLER${i}_SSH`. Not a blocker but should be addressed in a follow-up.